app/views/devise/confirmations/new.html.erb

```diff
@@ -1,42 +1,39 @@
 <% title "Confirm your email" %>
 <% proper_email = params[:email] || (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
-<main id="main-content">
+<main id="main-content" class="flex flex-1 justify-center flex-col crayons-layout">
   <% if FeatureFlag.enabled?(:creator_onboarding) %>
     <%= javascript_packs_with_chunks_tag "confirmationEmail", defer: true %>
-    <div class="crayons-layout crayons-layout--limited-l" id="email-confirmation">
-      <div class="flex flex-col items-center justify-center">
-        <%= render "devise/shared/error_messages", resource: resource %>
+    <div class="flex flex-1 flex-col items-center justify-center relative z-elevate align-center">
+      <%= render "devise/shared/error_messages", resource: resource %>
+      <%= inline_svg_tag("mail.svg", aria: true, title: "Email", class: "mb-6") %>
+      <h1 class="fs-2xl m:fs-3xl lh-tight fw-bold mb-4">Great! Now confirm your email address.</h1>
+      <% if proper_email.present? %>
+        <p class="fs-l m:fs-xl color-base-70 m:max-w-50">
+          We've sent an email to <span class="fw-bold"><%= proper_email %></span>.
+          Click the button inside to confirm your email.</p>
+        </p>
```

Comment:
Since this is now one paragraph with a span inside, the Cypress test is failing to match the text (it has difficulty verifying element text when there's a span or other element inside):

```
TestingLibraryElementError: Timed out retrying after 4000ms: Unable to find an element with the text: Click the button inside to confirm your email. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.

```

I don't think it's worth expending too much energy on - the test could be tweaked on line 19 to 

`cy.findByText(/Click the button inside to confirm your email/)`

https://github.com/forem/forem/blob/4337ce9d917856e326f965402c34cbd6f3976132/cypress/integration/creatorOnboardingFlows/confirmEmail.spec.js#L19

That should check for a partial match for the given text and pass OK.

Reply:
@ludwiczakpawel if you make this change, then the tests should pass and we're good to merge :)